### ISSUE TITLE
add privacy considerations section

### ISSUE THREAD

**Issue by samuelweiler** [w3c/mathml-core#49]
In Mathematical Markup Language (MathML) Version 4.0, separate from security considerations, per https://w3ctag.github.io/security-questionnaire/

**Comment by fred-wang** [comment]
@NSoiffer @davidcarlisle I believe this was originally for https://w3c.github.io/mathml/ and should not have been transferred to the mathml-core repo ?

( I still need to split the section for mathml-core, but that's already tracked by https://github.com/w3c/mathml-core/issues/48 )

**Comment by davidcarlisle** [comment]
agreed, given the corresponding core issue is already here. I'll move this one back to mathml(4)
